hc/accounts/tests/test_login.py

```diff
@@ -18,14 +17,14 @@ def test_it_sends_link(self):
 
         r = self.client.post("/accounts/login/", form)
         assert r.status_code == 302
-
-        ### Assert that a user was created
-
+        ### **Assert that a user was created
+        final_list = len(User.objects.all())
+        self.assertEqual(1, final_list)
         # And email sent
         self.assertEqual(len(mail.outbox), 1)
         self.assertEqual(mail.outbox[0].subject, 'Log in to healthchecks.io')
         ### Assert contents of the email body
-
+        self.assertIn("please open the link below:", mail.outbox[0].body)
```

Comment:
Good point @georgreen however, the correct status code is not sufficient on its own. That's why we need to assert the content as well...preferably the dynamic stuff unique to the every login email.